toolkit/tools/imagegen/attendedinstaller/views/installerview/installerview.go

```diff
@@ -63,15 +63,42 @@ func New(calamaresInstallFunc func()) *InstallerView {
 	if err != nil {
 		logger.Log.Debugf("Calamares not found, defaulting to terminal based installer")
 	} else {
-		logger.Log.Debugf("Calamares found, but the gui installer is temporarily disabled until stabilized")
-		// iv.installerOptions = append(iv.installerOptions, uitext.InstallerGraphicalOption)
+		iv.installerOptions = append(iv.installerOptions, uitext.InstallerGraphicalOption)
+
+		err = AssignDbusPermissions()
+		if err != nil {
+			logger.Log.Debugf("An error occured during reassignment of dbus permissions")
+		}
 	}
 
 	iv.needsToPrompt = (len(iv.installerOptions) != 1)
 
 	return iv
 }
 
+// This function is a workaround to deal with squashed,required permissions
+// within the iso_initrd environment. kpmcore requires that the
+// /usr/libexec/dbus-daemon-lauch-helper file have the group messagebus with
+// permissions 4750 in order to run properly. kpmcore is used by calamares to
+// power the partition module used by the gui installer to configure disk
+// selection.
+// Information on issue here: https://invent.kde.org/system/kpmcore/-/issues/15
```

Comment:
After searching a bit more for that error, I think this is not an issue related to `kpmcore` but attempts to run any daemon with `dbus-daemon-launch-helper`, when the owning group for that file is not `messagebus`. Similar threads related to this error but not to `kpmcore`:
- https://forums.gentoo.org/viewtopic-t-1079170-start-0.html,
- https://forums.freebsd.org/threads/gdbus-error-org-freedesktop-dbus-error-spawn-execfailed.91776/,
- https://forums.gentoo.org/viewtopic-t-1007656-start-0.html.

It looks like in our case it might be the other way around: something's trying to start a `kpmcore` service by running `dbus-daemon-launch-helper` and fails. Considering that, I think we should re-word this comment to mention what we know:
- `dbus-daemon-launch-helper` fails to run due to `/usr/libexec/dbus-daemon-launch-helper` not being owned by the `messagebus` group.
- This prevents from starting a `kpmcore` service.
- All user and group ownership information is lost when we boot from the initrd.
- It is unclear (at least to me) why it has to be specifically the `messagebus` grup to run `dbus-daemon-launch-helper`.
- I'd also provide a few more links to similar issues like your link and the ones in this comment.

Reply:
Comment updated with additional information and extra links.